.circleci/config.yml

```diff
@@ -365,10 +357,10 @@ jobs:
       # Bundler
       - restore_cache:
           keys: 
-            - v1-gem-cache-{{ checksum "Gemfile.lock" }}
+            - v2-gem-cache-{{ checksum "Gemfile.lock" }}
       - run: bundle install --clean --path vendor/bundle
       - save_cache:
-          key: v1-gem-cache-{{ checksum "Gemfile.lock" }}
+          key: v2-gem-cache-{{ checksum "Gemfile.lock" }}
```

Comment:
Hmm I wonder why we don't call `install_dependencies` here instead... But it's not related to this change, so we can look into it later.